tests/blackbox/stratis_cert.py

```diff
@@ -0,0 +1,347 @@
+        :return: None
+        """
+        pool_name = p_n()
+        StratisCli.pool_create(pool_name, [DISKS[0]])
+        StratisCli.pool_add(pool_name, "add-cache", DISKS[1:])
+        block_devs = StratisCli.blockdev_list()
+
+        for d in DISKS:
+            self.assertTrue(d in block_devs)
+
+    def test_fs_create(self):
+        """
+        Test creating a FS
+        :return: None
+        """
+        pool_name = p_n()
+        fs_name = fs_n()
+        StratisCli.pool_create(pool_name, [DISKS[0]])
+        StratisCli.fs_create(pool_name, fs_name)
+        fs = StratisCli.fs_list()
+        self.assertTrue(fs_name in fs.keys())
+        self.assertEqual(1, len(fs))
+        self.assertEqual(fs[fs_name]["POOL_NAME"], pool_name)
+
+    def test_fs_destroy(self):
+        """
+        Test destroying a FS
+        :return:
+        """
+        pool_name = p_n()
+        fs_name = fs_n()
+        fs_too = fs_n()
+        StratisCli.pool_create(pool_name, [DISKS[0]])
+        StratisCli.fs_create(pool_name, fs_name)
+        StratisCli.fs_create(pool_name, fs_too)
+        StratisCli.fs_destroy(pool_name, fs_name)
+
+        fs = StratisCli.fs_list()
+        self.assertEqual(1, len(fs))
+        self.assertFalse(fs_n in fs)
+        self.assertTrue(fs_too in fs)
+
+    def test_fs_snap_shot(self):
+        """
+        Test creating a FS snap shot.
+        :return: None
+        """
+        pool_name = p_n()
+        fs_name = fs_n()
+        fs_ss = fs_n()
+        StratisCli.pool_create(pool_name, [DISKS[0]])
+        StratisCli.fs_create(pool_name, fs_name)
+        StratisCli.fs_ss_create(pool_name, fs_name, fs_ss)
+        fs = StratisCli.fs_list()
+        self.assertTrue(fs_ss in fs)
+
+    def test_block_dev_list(self):
+        """
+        Test block device listing
+        :return:
+        """
+        pool_name = p_n()
+        StratisCli.pool_create(pool_name, [DISKS[0]])
+
+        block_devs = StratisCli.blockdev_list()
+        self.assertTrue(DISKS[0] in block_devs)
+        self.assertEqual(1, len(block_devs))
+        self.assertEqual(pool_name, block_devs[DISKS[0]]["POOL_NAME"])
+
+    @unittest.expectedFailure
+    def test_no_args(self):
+        """
+        Ref. https://github.com/stratis-storage/stratis-cli/issues/248
+        :return: None
+        """
+        exec_command([STRATIS_CLI], 2)
+        exec_command([STRATIS_CLI, "daemon"], 2)
+
+    def test_fs_rename(self):
+        """
+        Test renaming a FS
+        :return: None
+        """
+        pool_name = p_n()
+        fs_name = fs_n()
+        fs_new_name = fs_n()
+        StratisCli.pool_create(pool_name, [DISKS[0]])
+        StratisCli.fs_create(pool_name, fs_name)
+        StratisCli.fs_rename(pool_name, fs_name, fs_new_name)
+
+        fs = StratisCli.fs_list()
+        self.assertTrue(fs_new_name in fs.keys())
+        self.assertFalse(fs_name in fs.keys())
+        self.assertEqual(1, len(fs))
+
+    @unittest.expectedFailure
```

Comment:
This test should succeed, since the fix has been merged, so expectedFailure should be taken off.

Reply:
Until the correction gets packaged into a release this need to stay as is.

Reply:
Oh, right.